client/signup/steps/theme-selection/index.jsx

```diff
@@ -101,42 +123,27 @@ class ThemeSelectionStep extends Component {
 
 	render = () => {
 		const defaultDependencies = this.props.useHeadstart ? { themeSlugWithRepo: 'pub/twentysixteen' } : undefined;
+		const { translate } = this.props;
 
-		const pressableWrapperClassName = classNames( {
-			'theme-selection__pressable-wrapper': true,
-			'is-hidden': ! this.state.showPressable,
-		} );
-
-		const themesWrapperClassName = classNames( {
-			'theme-selection__themes-wrapper': true,
-			'is-hidden': this.state.showPressable,
-		} );
+		let headerText = translate( 'Choose a theme.' );
```

Comment:
I think we'd prefer to avoid the use of `let` in favor of `const` if possible. 

In this case:

```
const headerText = this.state.showPressable ? "...?" : "...!";
// etc.
```